examples/ontology-from-excel/README.md

```diff
@@ -0,0 +1,24 @@
+# Generate an ontology from excel
+
+This directory contains an example xlsx-file for how to document ontology entities (classes, object properties, annotation properties and data properties) in an Excel workbook.
+This workbook can then be used to generate a new ontology or update an already existing ontology with new entities (existing entities are not updated).
+
+Please refer to the (documentation)[https://emmo-repo.github.io/EMMOntoPy/latest/api_reference/ontopy/excelparser/] for full explanation of capabilities.
+
+The file onto.xlsx contains examples on how to do things correctly as well as incorrectly.
```

Comment:
```suggestion
The file `tool/onto.xlsx` contains examples on how to do things correctly as well as incorrectly.
```

Reply:
I guess that it would be helpful with a short explanation of what i is correct and what is incorrect.

Reply:
If I run excel2onto from `examples/ontology-from-excel/tool/`, I get

```
FileNotFoundError: [Errno 2] No such file or directory: '/home/friisj/prosjekter/EMMC/EMMO-python/examples/ontology-from-excel/tool/ontology.ttl'
```

Should I enter the `imported_onto/` subsubdirectory? If so, it should be documented.

Reply:
> 

There are comments in the xlsx file about how they are incorrect.